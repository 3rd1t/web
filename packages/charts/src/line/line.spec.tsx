import React from "react"
import renderer from "react-test-renderer"
// Required for jest, see https://jestjs.io/docs/en/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom
import "../../../../__mocks__/matchMedia"
import Line from "./line"

describe("Line", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Line
          data={[
            { time: "01/01/2020", value: 1 },
            { time: "02/01/2020", value: 2 },
          ]}
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
