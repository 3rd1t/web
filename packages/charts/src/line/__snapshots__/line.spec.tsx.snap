// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line renders correctly 1`] = `
<div
  id="chart"
/>
`;
